far/viewer.cpp

```diff
@@ -1889,6 +1919,26 @@ bool Viewer::process_key(const Manager::Key& Key)
 
 			return true;
 		}
+		case KEY_ALTLEFT: case KEY_ALTNUMPAD4:
+		case KEY_RALTLEFT: case KEY_RALTNUMPAD4:
+		{
+			if (m_BytesPerLine > s_BytesPerSeparator)
+			{
+				--m_BytesPerLine;
```

Comment:
Would be nice to change it by 16 bytes at a time as well. `Ctrl+Alt+Left` maybe?

Reply:
Nice, added.

Reply:
Cool! Thank you.